apps/wfo-ui-surf/jest.config.cjs

```diff
@@ -3,5 +3,5 @@ const base = require('@orchestrator-ui/jest-config/jest-base.config.js');
 module.exports = {
     ...base,
     name: 'wfo-ui-surf',
-    displayName: 'WFO-UI Tests',
+    displayName: 'Wfo-UI Tests',
```

Comment:
Hmm, relying on search and replace?

Reply:
Yes. But this renaming is still valid right. We apply the convention in displaynames to no?

Reply:
The convention should be in filenames, variable names, component names. Strings representing the app name should still be WFO i would say